cron/controller/Dockerfile

```diff
@@ -27,6 +26,7 @@ ARG TARGETARCH
 RUN CGO_ENABLED=0 make build-pubsub
 
 FROM gcr.io/distroless/base:nonroot@sha256:bc84925113289d139a9ef2f309f0dd7ac46ea7b786f172ba9084ffdb4cbd9490
+USER 0
```

Comment:
what's this needed for?

Reply:
So I'm running into a weird issue where Go is unable to open "cron/data/projects.csv" file due to permission error. According to Dockerfile reference, the default user for the `COPY` command is `0`. So I tried setting `USER` to `0` before `ENTRYPOINT` and that seems to solve this. Not sure if this is the right solution though. Any ideas?

Reply:
That's weird. How did it work before?

It does seem a bit hacky to set the user to root given that we're using a "nonroot" image. Can you try using another distroless image (not the "nonroot" one) and see if that fixes things?

Reply:
So this issue happened due to https://github.com/ossf/scorecard/pull/706. I only caught it now when writing these e2e tests. Before that we were embedding the file and not reading it through `os.Open`.

Will try the "nonroot" image and get back to you.

Reply:
Fixed it, thanks to @dlorenc. The permission of the file in the repo only allowed `root` to have read access. Updated that now.